bin/server.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+var corsHeaders = require('hapi-cors-headers')
+var Hapi = require('hapi')
+var path = require('path')
+
+var plugin = require('../index')
+
+var server = new Hapi.Server()
+server.connection({ port: 3000 })
+
+server.register({
+  register: require('inert')
+}, function (error) {
+  if (error) throw error
+})
+
+// serve demo app at route path
+server.route({
+  method: 'GET',
+  path: '/{path*}',
+  handler: {
+    directory: {
+      path: path.resolve(__dirname, '../demo')
+    }
+  }
+})
+
+server.register({
+  register: require('good'),
+  options: {
+    reporters: [{
+      reporter: 'good-console',
+      events: { error: '*', log: '*', request: '*', response: '*', wreck: '*' }
+    }]
+  }
+}, function (err) {
+  if (err) {
+    console.error(err)
+  }
+})
+
+server.register({
+  register: require('h2o2')
+}, function (error) {
+  if (error) throw error
+})
+
+server.register({
+  register: plugin
+}, function (error) {
+  if (error) throw error
```

Comment:
Yeah what @Charlotteis says. I personally like the short form better for `throw` and `return` statements, but too many people are confused by it, so let's stick with the `{}` brackets